src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -427,11 +441,7 @@ List<TestCase> assignCaseNumbersWhenCaseNumbersNotExist(
         .sorted(
             Comparator.comparing(
                 TestCase::getCreatedAt, Comparator.nullsFirst(Comparator.naturalOrder())))
-        .map(
-            testCase -> {
-              testCase.setCaseNumber(sequenceService.generateSequence(measureId));
-              return testCase;
-            })
+        .peek(testCase -> testCase.setCaseNumber(sequenceService.generateSequence(measureId)))
```

Comment:
@RohitKandimalla Isn't peek intended for debugging purpose?

Reply:
Yeah it should I guess Intelli J fixed the entire file. I will revert it

Reply:
I'm also curious why the map was replaced with peek.

Reply:
@RohitKandimalla Ok thanks a lot Rohit! Should we trust AI then? haha..